flood_adapt/object_model/scenario.py

```diff
@@ -16,7 +17,7 @@ class Scenario(IScenario):
 
     attrs: ScenarioModel
     direct_impacts: DirectImpacts
-    database_input_path: Union[str, os.PathLike]
```

Comment:
Why not keep the string option? You can always convert the string to a path object (I personally prefer Pathlib over os.PathLike) first thing in the function

Reply:
reverted this change